lib/PSR/Storage/SQLite.hs

```diff
@@ -16,11 +16,15 @@ import Cardano.Api (
     serialiseToRawBytes,
  )
 import Cardano.Api qualified as C
+import Cardano.Ledger.Plutus (ExUnits)
 
+import Data.Aeson qualified as Aeson
+import Data.ByteString.Lazy qualified as BL
 import Data.Functor ((<&>))
 import Data.Maybe (catMaybes, fromMaybe)
 import Data.Text (Text)
 import Data.Text qualified as T
+import Data.Text.Encoding qualified as T
```

Comment:
Not needed.

Reply:
Do you have a HLS running on your mind?
How do you even spot unused imports?